testsuite/libra-fuzzer/src/commands.rs

```diff
@@ -39,9 +39,7 @@ pub fn make_corpus(
             }
         };
 
-        // Use the SHA-1 of the result as the file name.
-        sha1.input(&result);
```

Comment:
What happened for this to go away? Does the fuzzer no longer need a uinque file name?

Reply:
gah, good catch, thank you